scripts/main.js

```diff
@@ -0,0 +1,45 @@
+function Book(title, author) { this.title = title; this.author = author; }
+class Library {
+  constructor() {
+    if (window.localStorage.getItem('stored') === null) {
+      window.localStorage.setItem('stored', JSON.stringify([]));
+    }
+    this.books = JSON.parse(window.localStorage.getItem('stored'));
+    this.booklist = document.getElementById('booklist');
+    document.getElementById('add').addEventListener('click', () => { this.add(); });
+    this.show();
+  }
+
+  add() {
+    const title = document.getElementById('title').value;
+    const author = document.getElementById('author').value;
+    const book = new Book(title, author);
+
+    this.books.push(book);
```

Comment:
- [ ] OPTIONAL
It will be a good practice to favor the use array methods such as `concat` over `put`, that doesn't mutate the array.